generators/app/templates/server/config/application.rb

```diff
@@ -7,14 +7,15 @@
 require "active_record/railtie"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-# require "action_view/railtie"
-# require "action_cable/engine"
+require "action_view/railtie"
+require "action_cable/engine"
 # require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require "rails/test_unit/railtie"
```

Comment:
test_unit y
action_cable very interesting
action_view i was thinking about emails, what are other options? How much overhead this generates?